ibis/expr/schema.py

```diff
@@ -90,12 +90,10 @@ def __getitem__(self, name: str) -> dt.DataType:
 
     def equals(self, other: Schema) -> bool:
         """Return whether `other` is equal to `self`.
-
         Parameters
         ----------
         other
             Schema to compare `self` to.
-
```

Comment:
I believe these newlines are necessary. In any case, I find it hard to read this all scrunched together.